### DISCUSSION TITLE
Error when deploying an Astro app to Vercel

### DISCUSSION THREAD

**Discussion by ymcheung** [vercel/vercel#8117] [Help]
Hello! I have been deploying an Astro app to Vercel.

It was all fine when I also just update content 3 days ago but got errors when deploying a preview today.

```
"@astrojs/preact": "^0.0.2",
"@astrojs/renderer-preact": "^0.4.0",
"astro": "^0.26.0",
```

The error message:
```
Error when evaluating SSR module /src/components/...Item.jsx:
--
Error: Cannot find module 'preact/jsx-runtime' imported from '/vercel/path0/src/components/...Item.jsx'
at viteResolve (/vercel/path0/node_modules/.pnpm/vite@2.9.12_sass@1.53.0/node_modules/vite/dist/node/chunks/dep-8f5c9290.js:50310:25)
```

![CleanShot 2022-07-09 at 11 42 22@2x](https://user-images.githubusercontent.com/4409977/178090302-b20da64c-583b-4b43-95df-efa53b22c411.jpg)

I didn't change anything in `package.json` nor update any code (just markdown content!).

Local build is successful:

![CleanShot 2022-07-09 at 11 49 17@2x](https://user-images.githubusercontent.com/4409977/178090423-8ac87aee-5b91-4ee9-a4c7-e4fb892ba237.jpg)

Then I changed Node version to `16` on Vercel, which is the same as my local dev environment, and still got the errors.

Thank you for the help!

**Comment by amyegan** [comment]
Hi @ymcheung. Sorry for the very late response. I hope you already found a solution to your problem, but I am happy to help if you still see this error.

**Reply by jasnsy** [reply]
I'm actually experiencing the same error at the moment

**Reply by ymcheung** [reply]
I upgraded to stable versions of Astro and the deployment was successful.
Had to rework a lot of code in Astro islands though.

**Comment by jasnsy** [comment]
What version of Astro are you using that worked?

**Reply by ymcheung** [reply]
I use Astro from 1.0.2 to 1.3.x and everything is fine.

**Comment by jasnsy** [comment]
is there an Astro version manager? I don't see a command to pick/change the
version in the CLI

**Reply by amyegan** [reply]
Astro has a [migration guide](https://docs.astro.build/en/migrate/) to walk you through the upgrade process if you were using an older version

**Comment by jasnsy** [comment]
I'm actually running the newest Astro version (1.4.4), is reverting to an
older version possible?
